router/http/src/main/scala/io/buoyant/router/Http.scala

```diff
@@ -32,6 +32,7 @@ object Http extends Router[Request, Response] with FinagleServer[Request, Respon
     val client: StackClient[Request, Response] = FinagleHttp.client
       .transformed(StackRouter.Client.mkStack(_))
       .transformed(_.replace(http.TracingFilter.role, http.TracingFilter.module))
+      .transformed(_.insertBefore(http.TracingFilter.role, DiscardProxySideEffects.module))
```

Comment:
Looks like I failed and put it in wrong place although you initially told me to put it onto the server side. %) Will fix, thanks.